fastly/tls_mutual_authentication.go

```diff
@@ -0,0 +1,212 @@
+package fastly
+
+import (
+	"fmt"
+	"reflect"
+	"strconv"
+	"time"
+
+	"github.com/google/jsonapi"
+)
+
+// TLSMutualAuthentication represents a mutual authentication certificate.
+type TLSMutualAuthentication struct {
+	Activations []*TLSActivation `jsonapi:"relation,tls_activations"`
+	CreatedAt   *time.Time       `jsonapi:"attr,created_at,iso8601"`
+	Enforced    bool             `jsonapi:"attr,enforced"`
+	ID          string           `jsonapi:"primary,mutual_authentication"`
+	Name        string           `jsonapi:"attr,name"`
+	UpdatedAt   *time.Time       `jsonapi:"attr,updated_at,iso8601"`
+}
+
+// ListTLSMutualAuthenticationsInput is used as input to the Client.ListTLSMutualAuthentication function.
+type ListTLSMutualAuthenticationsInput struct {
+	// Include is a comma-separated list of related objects to include (optional).
+	Include string
```

Comment:
Any particular reason why this is `string` and not `[]string`?

Reply:
In this particular instance, the API suggests there's only one permitted value (`tls_activations`). So although the documentation says comma-separated, that's not really true, but I have to presume the people who wrote the documentation were trying to be as 'future proof' as possible, so maybe we do the same and use a slice and join the strings for the user (just in case more permitted values are added in future).